### DISCUSSION TITLE
AP + Client (on board)

### DISCUSSION THREAD

**Discussion by pariesen** [community/community#78629] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello
I configured openplotter as described. The number of clients accessing openplotter simultaneously through the WIFI local (10.10.10.1) is limited to 2 clients. How can you explain this?
Thank you
Patrick

**Comment by queenofcorgis** [comment]
Hi @pariesen ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on OpenPlotter. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by pariesen** [reply]
Hello
Thank you for your answer, but I am a little bit confused. I am referring to the documentation in GitHUB:
[network/network_app.rst](https://github.com/openplotter/docs/tree/3.x.x)
Thank you for your support.
Regards
